### PR TITLE
Update create-a-custom-sensitive-information-type-in-scc-powershell.md

### PR DESCRIPTION
**Pull request by kellybos**
Addition of note about known extraction issue with no ETA for fix. Bug - 

https://o365exchange.visualstudio.com/IP%20Engineering/_workitems/edit/2068902

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/create-a-custom-sensitive-information-type-in-scc-powershell.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ See [Potential validation issues to be aware of](#potential-validation-issues-to
 
 For more information about the Boost.RegEx (formerly known as RegEx++) engine that's used for processing the text, see [Boost.Regex 5.1.3](https://www.boost.org/doc/libs/1_68_0/libs/regex/doc/html/).
 
+> [!NOTE]
+> If you use an ampersand character (&) as part of a keyword in your custom sensitive information type, please note that there is a known issue. You should add an additional term with spaces around the character to make sure that the character is properly identified. For example -
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-custom-sensitive-information-type-in-scc-powershell.md]:
```suggestion
> If you use an ampersand character (&) as part of a keyword in your custom sensitive information type, please note that there is a known issue. You should add an additional term with spaces around the character to make sure that the character is properly identified, for example, L & P _not_ L&P.
```

File: microsoft-365/compliance/create-a-custom-sensitive-information-type-in-scc-powershell.md [hunk 2]
```diff
@@ -34,6 +34,12 @@ See [Potential validation issues to be aware of](#potential-validation-issues-to
 
 For more information about the Boost.RegEx (formerly known as RegEx++) engine that's used for processing the text, see [Boost.Regex 5.1.3](https://www.boost.org/doc/libs/1_68_0/libs/regex/doc/html/).
 
+> [!NOTE]
+> If you use an ampersand character (&) as part of a keyword in your custom sensitive information type, please note that there is a known issue. You should add an additional term with spaces around the character to make sure that the character is properly identified. For example - 
+> 
+><Term>L&P</Term>
+><Term>L & P</Term>
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-custom-sensitive-information-type-in-scc-powershell.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@kellybos Please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks.

**Comment by kellybos** [comment]
@chrfox Any update on getting this PR finalized?

**Pull request merged**